renderers/server-render.jsx

```diff
@@ -0,0 +1,172 @@
+import Rx from 'rxjs';
```

Comment:
yes, that makes sense - I'll update this PR and the corresponding files in mup-web#312